etc/docs/template/layouts/partials/assets/analytics.html

```diff
@@ -0,0 +1,9 @@
+<script>
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+  ga('create', 'UA-29229787-1', 'auto');
```

Comment:
Can we find out who has access to this reporting, we should double check this goes somewhere useful.

Reply:
Yeah, I can check

Reply:
Mary confirmed that we do intentionally collect these analytics